### DISCUSSION TITLE
OneBlock - Phase

### DISCUSSION THREAD

**Discussion by Idriss60** [BG-Software-LLC/SSB-OneBlock#41] [General]
I find SuperiorSkyblock2 much more complete than Bentobox. However, the OneBlock addon can still be improved. Looking at it, the main thing that bothered me was the phase configuration. It is abominable. I suggest to take inspiration from the Bentobox configuration and use % instead. It would make our job so much easier 🙏

**Comment by OmerBenGera** [comment]
Mind giving more details on how it can be improved?

**Reply by Idriss60** [reply]
Can we talk on discord to facilitate the exchange?

**Reply by OmerBenGera** [reply]
> Can we talk on discord to facilitate the exchange?

Sure, ping me on the discord server and we'll talk there

**Reply by Idriss60** [reply]
Okay
